ArticleTemplates/assets/scss/modules/_alerts.scss

```diff
@@ -43,40 +43,57 @@
     transition: background-color .3s;
     transform: translateX(-1px);
 
-    span[data-icon] {
-        display: inline-block;
-        vertical-align: middle;
-        border-radius: 100%;
-
-        width: 24px;
+    &__icon_background, &__icon_background_full {
         height: 24px;
-        line-height: 20px;
+        width: 24px;
+        border: currentColor 1px solid;
+        border-radius: 50%;
+        display: inline-block;
+        background-color: none;
+    }
+    &__icon_background_full {
         background-color: currentColor;
-        text-align: center;
-        font-size: 15px;
-        &:before {
-            color: color(brightness-100);
-            padding-top: 1px;
-            display: block;
-        }
+        z-index: -1;
+    }
+    svg {
+        position: absolute;
+        left: 5px;
+        top: 6px;
+        height: 15px;
+        width: 14px;
+        fill: currentColor;
+        fill-opacity: 1;
     }
-
     &__label {
         font-weight: normal;
-        margin-left: 2px;
+        margin-left: 4px;
+        font-size: 15px;
+        color: color(brightness-0);
     }
-
-    &__state--follow {
+    &__state--follow  {
         display: none;
     }
 
-    &.following {
-        .alerts__state--follow {
-            display: block;
+    &__state--follow, &__state--notifications-on {
+        svg {
+           fill: white;
```

Comment:
I think you need to use the article background colour here. not always white. in fact, it might never be actually white.

Reply:
This will be apparent in Comments which are pinkish and maybe darkmode?

Reply:
The colour palette system here is completely different to Source but imagine can match by the hex code. 

These are the colour files:
https://github.com/guardian/mobile-apps-article-templates/blob/5357690df3b8479b7bbeec8d35f102365ba11625/ArticleTemplates/assets/scss/base/_colour.scss

It looks to me like all the colours are being mixed in by class name via specific colour stylesheets:
https://github.com/guardian/mobile-apps-article-templates/tree/5357690df3b8479b7bbeec8d35f102365ba11625/ArticleTemplates/assets/scss/pillar
Darkmode themes:
https://github.com/guardian/mobile-apps-article-templates/tree/5357690df3b8479b7bbeec8d35f102365ba11625/ArticleTemplates/assets/scss/themes/darkMode
Special Tones:
https://github.com/guardian/mobile-apps-article-templates/blob/5357690df3b8479b7bbeec8d35f102365ba11625/ArticleTemplates/assets/scss/tones/_tone--special.scss